<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>options</string>
  </property>
  <widget class="QPushButton" name="startgame_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 237);
color: rgb(25, 43, 194);
font: 22pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>START GAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highscore_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 237);
color: rgb(25, 43, 194);
font: 16pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>HIGHSCORE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="online_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 237);
color: rgb(25, 43, 194);
font: 16pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>ONLINE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signout_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 237);
color: rgb(25, 43, 194);
font: 20pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>SIGN OUT</string>
   </property>
  </widget>
  <widget class="QListWidget" name="highscore_list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="online_list">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="highscore_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>39</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>72</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>22</number>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>score</string>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <zorder>startgame_button</zorder>
  <zorder>signout_button</zorder>
  <zorder>highscore_list</zorder>
  <zorder>online_list</zorder>
  <zorder>online_button</zorder>
  <zorder>highscore_table</zorder>
  <zorder>highscore_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
